climada/entity/exposures/base.py

```diff
@@ -415,7 +417,9 @@ def plot_hexbin(self, mask=None, ignore_zero=False, pop_name=True,
             mask (np.array, optional): mask to apply to eai_exp plotted.
             ignore_zero (bool, optional): flag to indicate if zero and negative
                 values are ignored in plot. Default: False
-            pop_name (bool, optional): add names of the populated places
+            pop_name : bool or str, optional
+                add names of the populated places, by default True. If set to 'markers' only the points
+                will be shown without a description.
```

Comment:
Probably it is better then to change to `,None', 'names', 'markers'` or something along these lines. Mixing types  in variables is not a  very good idea most of the time.